src/styles/annotator/highlights.scss

```diff
@@ -1,35 +1,70 @@
 :root {
+  // Some variables here are camel-cased for ergonomic convenience when
+  // manipulating with JavaScript
+
   // Default highlight styling configuration
   --hypothesis-highlight-color: rgba(255, 255, 60, 0.4);
+  --hypothesis-highlight-secondColor: rgba(206, 206, 60, 0.4);
+  --hypothesis-highlight-thirdColor: transparent;
+
   --hypothesis-highlight-focused-color: rgba(156, 230, 255, 0.5);
-  --hypothesis-highlight-blend-mode: normal;
-  --hypothesis-highlight-decoration: none;
   --hypothesis-highlight-text-color: inherit;
 
-  --hypothesis-highlight-second-color: rgba(206, 206, 60, 0.4);
-  --hypothesis-highlight-third-color: transparent;
+  --hypothesis-color-cyan: #cffafe;
 
   // Colors available for clustered highlights
-  --hypothesis-color-blue: #e0f2fe;
-  --hypothesis-color-yellow: #fef9c3;
-  --hypothesis-color-purple: #ede9fe;
-  --hypothesis-color-orange: #ffedd5;
-  --hypothesis-color-green: #d1fae5;
+  // Base colors sourced from https://tailwindcss.com/docs/customizing-colors
+
+  // Stone 100, 200 and 300
   --hypothesis-color-grey: #f5f5f4;
-  --hypothesis-color-pink: #ffe4e6;
+  --hypothesis-color-grey-1: #e7e5e4;
+  --hypothesis-color-grey-2: #d6d3d1;
+
+  // Yellow 100
+  --hypothesis-color-yellow: rgb(254, 249, 195);
+  // Hand-adjusted
+  --hypothesis-color-yellow-1: rgb(253, 243, 149);
+  --hypothesis-color-yellow-2: rgb(252, 235, 111);
+
+  // The second and third colors for the following are computed by:
+  // - color-1: multiply base color @ 80% alpha on white
```

Comment:
Can you expand on this step?

Reply:
I find I'm falling into a bit of a rabbit hole trying to unpack this usefully right at the moment. There absolutely is a way to compute alpha-influenced colors (https://stackoverflow.com/questions/746899/how-to-calculate-an-rgb-colour-by-specifying-an-alpha-blending-amount e.g.) but my methodology to this point has merged math and manual:

1. Multiply the base color by itself. I did this on RGB channels, for each channel: `channel * channel / 255`
2. Figure out full-opacity color corresponding to `rbga(multipliedR, multipliedG, multipledB, 0.8);`. I confess I've been using manual color tools for this step to this point (eye-droppering the color out of the browser). 

Is there a way I could express this quickly in a comment without having to invest too much effort in the specifics of arithmetic?

Part of the reason I'm so avoidant of dropping time into this is I believe these colors—and thus the mechanism for calculating them—are likely to change once I _do_ focus on them.

Reply:
OK, thanks for the info. I think that's enough context for now and we can go ahead and get this merged. It isn't critical to have some functional expression for how these colors are computed, merely an "it might be useful in future".